src/components/layouts/footer/LayoutFooter.tsx

```diff
@@ -7,7 +7,20 @@ import { Layout } from 'antd';
 import { footerText } from '../../utils/constants';
 
 function LayoutFooter(): JSX.Element {
-    return <Layout.Footer>{footerText}</Layout.Footer>;
+    return (
+        <Layout.Footer
+            style={{
```

Comment:
style must be moved to css file. 
Also the color of the footer has now changed. You can compare with the existing  behavior on the demo environment.